### DISCUSSION TITLE
Importing Tags

### DISCUSSION THREAD

**Discussion by wz4** [netbox-community/netbox#9154] [General]
Version 3.2. I'm having an impossible time importing tags. I get an error that states to enter a list of values. Can someone please share the proper format to import tags via csv? Thanks.

Edit: The tags are already created, I am encountering this error while importing devices and vlans.

**Comment by candlerb** [comment]
Can you show a sample of the CSV you're trying to import? (Header line plus one line of data)

**Reply by wz4** [reply]
This is a vlan import. The error I receive is "Row 1 tags: enter a list of values"

```
vid,name,status,tags
10,test,active,"Test VLAN"
```

**Comment by jeremystretch** [comment]
Assigning tags during CSV import isn't actually supported yet; see #7376. It looks like the `tags` field was mistakenly added to this (and other) import forms during some of the refactoring for v3.2. I've opened #9158 to address this. Thanks for the report!
